app/queries/progress_reports/activity_session.rb

```diff
@@ -5,7 +5,7 @@ def initialize(teacher)
 
   def results(filters)
     filters = (filters || {}).with_indifferent_access
-    query = ActivitySession.includes(:user, activity: [:topic, :classification], classroom_activity: :classroom)
+    query = ActivitySession.includes(:user, :classroom_activity, activity: [:topic, :classification])
```

Comment:
Marcello handled these queries, but I think it is just working around not being able to grab classrooms in the same way